api/channel.go

```diff
@@ -570,6 +570,11 @@ func getChannelExtraInfo(c *Context, w http.ResponseWriter, r *http.Request) {
 		channel := cresult.Data.(*model.Channel)
 		member := cmresult.Data.(model.ChannelMember)
```

Comment:
we're still loading all the channel members?  I wonder if the load should be moved after the handleEtag since 90%+ of the time it will not change and no need to load the members again?